_episodes_rmd/04-ggplot2.Rmd

```diff
@@ -341,7 +341,7 @@ interviews_plotting %>%
 By adding points to a boxplot, we can have a better idea of the number of
 measurements and of their distribution:
 
-```{r boxplot-with-jitter, purl=FALSE}
+```{r boxplot-with-jitter, fig. alt = "Previous plot with dot plot added as additional layer to show individual values. Boxplot layer is transparent." purl=FALSE}
```

Comment:
```suggestion
```{r boxplot-with-jitter, fig. alt = "Previous plot with dot plot added as additional layer to show individual values. Boxplot layer is transparent.", purl=FALSE}
```